internal/mode/static/state/graph/graph.go

```diff
@@ -183,6 +185,15 @@ func BuildGraph(
 
 	npCfg := getNginxProxy(state.NginxProxies, processedGwClasses.Winner)
 	gc := buildGatewayClass(processedGwClasses.Winner, npCfg, state.CRDMetadata, validators.GenericValidator)
+	if gc != nil && npCfg != nil {
+		for _, cond := range gc.Conditions {
```

Comment:
Boolean probably won't work, because if paramsRef doesn't exist, what would the value be? We would need an enum for the 3 different states. Doesn't exist, exists and is invalid, and exists and is valid.

With how few conditions we have to loop through, I don't think there's necessarily much cost here.